src/shared/icon/icon.css.ts

```diff
@@ -4,6 +4,7 @@ const styles = css`
   :host {
     display: flex;
```

Comment:
Graag advies. Wat is de standaard werkwijze?

Elke component heeft eigen variabelen die gebaseerd zijn op thema variabelen zodat je die kan overschrijven (één van beide of beiden).

Of component gebruikt rechtstreeks thema variabelen?

Het gaat nu een beetje door elkaar heen voor mijn gevoel.

Reply:
Het is altijd afhankelijk van wie de verantwoordelijke/eigenaar is van een instelling. Zie ook https://github.com/biz-dev-ops/web-components/pull/368#discussion_r2025326128